app/controllers/api/v1/games_controller.rb

```diff
@@ -8,6 +8,7 @@ def create
         render json: questions, status: :internal_server_error
       else
         Rails.cache.write(game.id, questions, expires_in: 1.hour)
+        broadcast_players_and_status_for(game)
         render json: GameSerializer.format(game, questions), status: :created
       end
```

Comment:
This is incredibly cool how we can get to format the responses with our serializer!